flaml/nlp/README.md

```diff
@@ -0,0 +1,10 @@
+How to run coverage report:
```

Comment:
The content of this file needs to be changed to introduce how to use this package.